eval/README.md

```diff
@@ -0,0 +1,26 @@
+# Evaluation
+
+### Running HEST
+
+1. Install HEST and get the HEST-1k data according to the HEST's manual
+2. Checkout the submodules `git submodule update --init --recursive`
+3. Install hestcore `pip install hestcore`
+4. Run the benchmark: `python run_hest.py --config hest_bench_config.yaml`
+
+### Running EVA
+1. Checkout the submodules `git submodule update --init --recursive`
+2. Copy the custom model definitions [kaiko.py](./eva/src/eva/vision/models/networks/backbones/pathology/kaiko.py) and other helpers to eva: `cp kaiko.py object_tools.py backbones.py renormalized.py ./eva/src/eva/vision/models/networks/backbones/pathology/; mv kaiko.py kaiko.py_;`
```

Comment:
I suppose we can remove the `mv kaiko.py kaiko.py_` if you put all the eva patches in a folder called eva_patches, and copy from there.